src/components/navigation/HeaderTextButton.js

```diff
@@ -45,6 +56,14 @@ const getStyles = cacheStyles((theme: Theme) => ({
     paddingLeft: theme.rem(1),
     paddingRight: theme.rem(2.5),
     height: 44 // This is a fixed height of the navigation header no matter what screen size. Default by router-flux
+  },
+  left: {
```

Comment:
Padding left 1 and right 2.5 is already the default, so you don't need another style for 'left'.
Just use left as the default placement.

Reply:
Remove the default on container